src/Geometry/MeshReader.h

```diff
@@ -144,6 +158,29 @@ class MeshReader
   // scalingMatrix is stored column-major, i.e.
   // scalingMatrix_ij = scalingMatrix[j][i]
   void scaleMesh(double const scalingMatrix[3][3])
+  {
+    for (unsigned vertexNo = 0; vertexNo < m_vertices.size(); ++vertexNo) {
+      double x = m_vertices[vertexNo].coords[0];
+      double y = m_vertices[vertexNo].coords[1];
+      double z = m_vertices[vertexNo].coords[2];
+      for (unsigned i = 0; i < 3; ++i) {
+        m_vertices[vertexNo].coords[i] = scalingMatrix[0][i] * x + scalingMatrix[1][i] * y + scalingMatrix[2][i] * z;
+      }
```

Comment:
Scaling makes sense (e.g. to switch to a coordinate system where wavespeeds are in km/s or similar), but rotations etc. are a bit weird. I'm not sure when this would be useful. @Thomas-Ulrich any idea?

Reply:
I never used that and I don't know anybody who used it. But probably it was useful at some point if somebody implemented it :-) Any idea @AliceGabriel ?